privacyidea/lib/importotp.py

```diff
@@ -615,6 +616,7 @@ def export_pskc(tokenobj_list, psk=None):
         issuer = "privacyIDEA"
         try:
             manufacturer = tokenobj.token.description.encode("ascii")
+            manufacturer = to_unicode(manufacturer)
```

Comment:
We could also use the ``ignore`` option to just delete non-ASCII characters:
```
>>> 'Hællöhe'.encode('ascii', 'ignore')
b'Hllhe'
```
What do you think?

Reply:
will check.

Reply:
done in d367ea32